pages/Addons.md

```diff
@@ -74,6 +74,7 @@ SlimefunAdvancements | char321 | <details><summary>expand</summary>SlimefunAdvan
 Supreme | RelativoBR | <details><summary>expand</summary>Supreme is an addon for Slimefun which adds 100+ various new resources that will allow you to craft powerful new items, weapons, tools and armor. These can be made up from titanium, aurum, adamantium, thornium with some being magical, rare, epic, legendary or supreme! It also adds 12 new electric generators, 5 new capacitors and even more... </details> | [Readme](https://github.com/RelativoBR/Supreme/blob/main/README.md) | [Github](https://github.com/RelativoBR/Supreme) | [Builds](https://thebusybiscuit.github.io/builds/RelativoBR/Supreme/main/)
 Bump | SlimefunGuguProject | <details><summary>expand</summary>A Slimefun addon that adds magic weapons, food and an appraisal system. Through the appraisal system, you could even get an 8 star weapon or armor.</details> | [Readme](https://github.com/SlimefunGuguProject/Bump/blob/main/README.md) | [Github](https://github.com/SlimefunGuguProject/Bump) | [Builds (via Third-Party site)](https://builds.guizhanss.net/SlimefunGuguProject/Bump/main/)
 Netheopoiesis | Sefiraat / J3fftw | <details><summary>expand</summary>This addon was our entry into the Slimefun Addon Jam 2022. The premise of the addon is to allow players to play vanilla Minecraft and Slimefun solely within the Nether. You can purify and beautify the Nether over time including changing biomes, spawns and lots lots more. This is held up by a robust plant growing/breeding and harvesting system with plants that fullfill many functions including activities like spawning mobs, growing trees, dropping items and more. A branching but linear progression through the addon should provide lots of content, with more to come.</details> | [Readme](https://github.com/Sefiraat/Netheopoiesis/blob/master/README.md) | [Github](https://github.com/Sefiraat/Netheopoiesis) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/Netheopoiesis/master/)
+Relics of Cthonia | FN_FAL113 | <details><summary>expand</summary>An entry for addon jam 2022 which introduces relics in the nether dimension. These relics can be mined or unearthed from blocks and also can drop from mobs. One can trade these relics to piglins and receive awesome rewards! Be part of a new adventure on seeking the path to fortune by discovering these relics!</details> | [Readme](https://github.com/FN-FAL113/RelicsOfCthonia/blob/main/README.md) | [Github](https://github.com/FN-FAL113/RelicsOfCthonia) | [Builds](https://thebusybiscuit.github.io/builds/FN-FAL113/RelicsOfCthonia/main/)
```

Comment:
'and also can drop from mobs' Either 'and also drop from mobs' or 'can also drop from mobs' Otherwise lgtm

Reply:
Changed, appreciate that